### ISSUE TITLE
Auto-Add Suffix To Import

### ISSUE THREAD

**Issue by Skillz4Killz** [denoland/vscode_deno#138]
**Is your feature request related to a problem? Please describe.**
It is quite annoying having to manually add the `.ts` suffix to the imports.

When you auto-import a local file:
![image](https://user-images.githubusercontent.com/23035000/89946110-6fa5eb00-dbf0-11ea-816d-4bd3916fc6b2.png)

![image](https://user-images.githubusercontent.com/23035000/89946135-77fe2600-dbf0-11ea-8249-8be3babf850c.png)

Now when you are coding something 100 lines into the file and auto-import something, it's really annoying to have to go to the top of the page to fix the import because without the proper suffix i don't have access to any of the auto-completion of what i imported. It won't show any properties or methods. This is super annoying.

**Describe the solution you'd like**

Perhaps a config option in .vscode/settings.json thaat could be like `deno.addSuffixToLocalImport: ".ts"`

**Comment by nayeemrmn** [comment]
More precisely, it should _include_ a file extension in the auto-added import statements that TS intellisense supports. There doesn't need to be a setting for it, that should just be the behaviour where vscode_deno is active.

**Comment by axetroy** [comment]
Ther is no need to add a new option

autocomplete behavior is defined in here:

https://github.com/denoland/vscode_deno/blob/9d121207d8c1b9b9ad86a5b441d1bc674ca3fc14/typescript-deno-plugin/src/plugin.ts#L307-L381

You can do a PR to try to fix it when others have no time to do this.

**Comment by Skillz4Killz** [comment]
Been trying all morning to get this to work but sadly I am unable to get it to work.

I can't even get it to read the `plugin.ts` file. I've added breakpoints which do nothing then I added console.logs which did nothing. Finally, I added a console.log at the top of the file just to see if that file was even processed and it logged nothing. 

Anything I do in this file seems to have no effect when I am debugging.

I also encountered some other issues which I am not sure is the cause like this 
![image](https://user-images.githubusercontent.com/23035000/90024385-ccea7c80-dc82-11ea-983f-5bb488c5a503.png)

I use Mac not sure if that makes any difference.

**Comment by axetroy** [comment]
@Skillz4Killz `console.log` not work in `typescript-deno-plugin`

use `this.logger` in `plugin.ts`

and then run command in vscode `open TS server log`

**Comment by Skillz4Killz** [comment]
Been at this for the last couple of hours and can not for the life of me solve this. I really don't understand how people can code a VSC extension when I can't even have any debugger or logging working at all and the logger.info is just a ton of useless data to comb through just to figure it out and almost 4 hours later still can't manage to tell if this file is even running.

Being unable to tell if a file was even running, I've tried modifying almost every single line that mentioned something about a filepath like
```ts
resolvedFileName: v.extension === '.ts' ? `${v.filepath}.ts` : v.filepath,
```
 and it does basically nothing different. 

Sorry, I tried but i dont think I will be able to solve this. We need better ways to logging or debugging to help identify issues for those who don't code VSC extension but want to help contribute a small fix/enhancement here or there.
